http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/ProxyRoutes.scala

```diff
@@ -237,17 +251,29 @@ class ProxyRoutes(proxyService: ProxyService, corsSupport: CorsSupport, refererC
     for {
       ctx <- ev.ask[AppContext]
       apiCall = proxyService.proxyRequest(userInfo, googleProject, runtimeName, request)
-      _ <- metrics.incrementCounter("proxyRuntime")
       resp <- ctx.span.fold(apiCall)(span => spanResource[IO](span, "proxyRuntime").use(_ => apiCall))
+
+      headerMap: Map[String, String] = request.headers.map(header => (header.name(), header.value())).toMap
+      contentLength = Try(headerMap.get("content-length").map(_.toDouble)).toOption.flatten
+
+      _ <- if (request.uri.toString.endsWith(".ipynb") && request.method.equals(Method.PUT)) {
```

Comment:
I think `Method.PUT` is an http4s object.. probably want `HttpMethods.PUT` which is akka-http.

Also in Scala generally use `==` instead of `.equals` -- unlike Java this tests object equality and is more type-safe.